src/styles/main.scss

```
@@ -1,3 +1,200 @@
-body {
+body,
+h2,
+p {
   margin: 0;
 }
+
+a {
```

you can this style fo all elements
```suggestion
* {
```